### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by dorosiemers**

### CODE REVIEW

**Comment by kornrumpfl** [comment]
LGTM! 
I think is something wrong with the logo.png file!

**Review by kornrumpfl — APPROVED** [review]

File: pages/404.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        Hier geht es zurück zur{" "}
+        <Link href="/">
+          <a>Homepage</a>
+        </Link>
+      </p>
+    </div>
+  );
+};
+
+export default NotFound;
```
Comment (mpagels) [inline pages/404.js:18]: Love it. Very good thinking to add a 404 page.

**Review by mpagels — APPROVED** [review]
Very good work here Doro.

I recommend adding a short description for the reviewer, what is happening here in the pull request. :-) 

I saw that you added `mui` to the project. Here were advise you, not to use it. Because you should learn to build stuff by yourself. (if you need a complex date picker, it might better to use mui, because they solved the problem already. but for simple components you should try it by yourself. That said, I really don't know what your attention was with `mui`. I'm just saying the stuff in advance. :-)

Again, very good work here Doro. Keep up.

**Comment by dorosiemers** [comment]
> Very good work here Doro.
> 
> I recommend adding a short description for the reviewer, what is happening here in the pull request. :-)
> 
> I saw that you added `mui` to the project. Here were advise you, not to use it. Because you should learn to build stuff by yourself. (if you need a complex date picker, it might better to use mui, because they solved the problem already. but for simple components you should try it by yourself. That said, I really don't know what your attention was with `mui`. I'm just saying the stuff in advance. :-)
> 
> Again, very good work here Doro. Keep up.

Thanks Martin! I downloaded it and then realized we're not supposed to use it. That's all 👯‍♀️

**Comment by mpagels** [comment]
> > Very good work here Doro.
> > I recommend adding a short description for the reviewer, what is happening here in the pull request. :-)
> > I saw that you added `mui` to the project. Here were advise you, not to use it. Because you should learn to build stuff by yourself. (if you need a complex date picker, it might better to use mui, because they solved the problem already. but for simple components you should try it by yourself. That said, I really don't know what your attention was with `mui`. I'm just saying the stuff in advance. :-)
> > Again, very good work here Doro. Keep up.
> 
> Thanks Martin! I downloaded it and then realized we're not supposed to use it. That's all 👯‍♀️

Alright. Maybe you find the command in the `npm` docs to uninstall packages you don't need. That keeps the `node_modules` folder clean and small.

**Pull request merged**